ffi/capi/src/pluralrules.rs

```diff
@@ -138,6 +140,15 @@ pub mod ffi {
         pub fn from_fixed_decimal(x: &crate::fixed_decimal::ffi::Decimal) -> Box<Self> {
             Box::new(Self((&x.0).into()))
         }
+
+        #[diplomat::rust_link(icu::plurals::PluralOperands::is_exactly_one, FnInStruct)]
+        pub fn is_exactly_one(&self) -> bool {
+            self.0.is_exactly_one()
+        }
+        #[diplomat::rust_link(icu::plurals::PluralOperands::is_exactly_zero, FnInStruct)]
+        pub fn is_exactly_zero(&self) -> bool {
+            self.0.is_exactly_zero()
+        }
```

Comment:
I think these should actually not be pub in Rust, nothing else is exposed on `PluralOperands`: https://unicode-org.github.io/icu4x/rustdoc/icu/plurals/struct.PluralOperands.html. We needed them for `PluralElements` when that was living in `icu_experimental`, but it's in `icu_plurals` now.